.circleci/config.yml

```diff
@@ -75,12 +77,12 @@ commands:
             HOMEBREW_NO_AUTO_UPDATE=1 brew install applesimutils
 
   setup-e2e-creds:
-    description: "Decode account credentials for e2e tests"
+    description: "Generate an access token for e2e tests"
     steps:
       - run:
-          name: "Decode creds"
+          name: "Generate access token"
           command: |
-            echo "export const token ='$(curl $ACCESS_TOKEN_URL)'" >> test/TwilioVoiceReactNativeExample/src/e2e-tests-token.ts
+            node test/scripts/gen-token.mjs id-$CIRCLE_JOB-$CIRCLE_WORKFLOW_JOB_ID > test/TwilioVoiceReactNativeExample/src/e2e-tests-token.ts
```

Comment:
Since this is now a script, it's better to pass a filepath parameter and write to that file.